test/api/client/AuthenticationApi.ts

```diff
@@ -23,7 +24,7 @@ export default class AuthenticationApi {
     // Send
     const response = await this._baseApi.send({
       method: 'POST',
-      url: '/client/auth/Login',
+      url: '/v1/auth/' + ServerAction.REST_SIGNIN,
```

Comment:
Use full path

Reply:
What do you mean by full path ?

Reply:
I do not see what can be done better here at the URI path management.